### PR TITLE
fix: bring and fix some old test

### PR DESCRIPTION
**Pull request by yihong0618**
limited
1.  one test based on data-science-bundle
2. one test need logging but docker in docker seems do not have that.

```log
+DETAIL:  Failed to start container 7fa130c014be4f5fe6fe903463f2eee825d360f91c007b2b2473e74d975165de, return code: 500, detail: {"message":"failed to initialize logging driver: journald is not enabled on this host"}
```

### CODE REVIEW

**Review by Sasasu — APPROVED** [review]

**Comment by Sasasu** [comment]
> journald is not enabled on this host

CI runs in docker in docker. the first docker does not have journald,

document on docker official web https://docs.docker.com/config/containers/logging/local/

**Comment by yihong0618** [comment]
> > journald is not enabled on this host
> 
> CI runs in docker in docker. the first docker does not have journald,
> 
> document on docker official web https://docs.docker.com/config/containers/logging/local/

Thanks~

File: tests/regress/sql/user_control_py3.sql [hunk 1]
```diff
@@ -5,31 +5,31 @@ return math.log10(100)
...
+\! plcontainer runtime-add -r plc_python_user -i python39.alpine -l python3 -s use_container_logging=yes -s roles=gpadmin;
 SELECT plcontainer_refresh_local_config(false);
 -- end_ignore
 
 -- should success
 SELECT pylog100();
 
 -- start_ignore
-\! plcontainer runtime-replace -r plc_python_user -i pivotaldata/plcontainer_python3_shared:devel -l python3 -s use_container_logging=yes -s roles=plc1;
+\! plcontainer runtime-replace -r plc_python_user -i python39.alpine -l python3 -s use_container_logging=yes -s roles=plc1;
```
Comment (beeender) [inline tests/regress/sql/user_control_py3.sql:16]: If we hardcode this to alpine, will #646 work?

Comment (yihong0618) [reply tests/regress/sql/user_control_py3.sql:16]: yes, all changed to variable

**Review by beeender — APPROVED** [review]

**Pull request merged**